CRM/Upgrade/Snapshot.php

```diff
@@ -44,8 +44,20 @@ class CRM_Upgrade_Snapshot {
   public static function getActivationIssues(): array {
     if (static::$activationIssues === NULL) {
       $policy = CRM_Utils_Constant::value('CIVICRM_UPGRADE_SNAPSHOT', 'auto');
+      static::$activationIssues = [];
+
+      $version = CRM_Utils_SQL::getDatabaseVersion();
+      if (stripos($version, 'mariadb') !== FALSE) {
+        // MariaDB briefly (10.6.0-10.6.5) flirted with the idea of phasing-out `COMPRESSED`. By default, snapshots won't work on those versions.
+        // https://mariadb.com/kb/en/innodb-compressed-row-format/#read-only
+        $roCompressed = CRM_Core_DAO::singleValueQuery('SELECT @innodb_read_only_compressed');
```

Comment:
Should be `SELECT @@innodb_read_only_compressed` (note the two at signs).

A quick test for me shows a value of `0` with it properly turned off so we can create those snapshots (at least on MariaDB 10.6.3). So, the check for `1` should work.